server/controllers/chartsController/helpers.test.js

```diff
@@ -1009,4 +1009,91 @@ describe('chartsController - helpers', () => {
       ])
     })
   })
+
+  describe(helpers.graphTableColumns, () => {
+    it('returns an array of table headers from a list of labels', () => {
+      expect(helpers.graphTableColumns(helpersFactories.labels)).toEqual([
+        'Site',
+        'Pending evaluation',
+        'Excellent',
+        'Good',
+        'Average',
+        'Poor',
+        'Total',
+      ])
+    })
+
+    it('removes the N/A category from the labels list', () => {
+      const isNAValueInArray = helpers
+        .graphTableColumns(helpersFactories.labels)
+        .some((label) => label === N_A)
```

Comment:
we should set up the fixture data to have N/A data in this test so we know it's there even if the fixture changes to not have it anymore.

Reply:
Ya, good point. If folks don't see that NA data being added then it wouldn't make sense.
https://github.com/AMP-SCZ/dpdash/pull/287/commits/53c75c19215f81ea1f646fb5e98102019a21521f